### ISSUE TITLE
Add customizable text module to the confirmation email

### ISSUE THREAD

**Issue by dkehne** [digitalfabrik/entitlementcard#1757]
**Is your feature request related to a problem? Please describe.**
Applicants quickly become impatient if no progress is made on their application. In some cities and districts, there is only one person who processes the applications. However, this person is sometimes sick or on vacation. Especially then, a notice should be sent to all applications received from then on.

**Describe the solution you'd like**
In the confirmations of receipt sent to applicants after the application has been sent, a different absence message for each region should be inserted as a text module in a position to be defined. 
The text message is maintained in the region settings for each region. If the input field is not filled in, the confirmation of receipt remains as before. If a text is inserted, e.g. “There is currently a processing time of 4-6 weeks due to the Christmas period”, this text is automatically added to all outgoing confirmation emails from this region.

Update:
- It may make more sense to have a textfield where the user can type the absence message and a checkbox that can be enabled and disabled. Often the same or similar message will be reused

Acceptance Criteria:
- create a section in the region settings with a textfield for the absence message and a checkbox that indicates whether its enabled or disabled
- update the application confirmation mail for the applicant and set this as a last sentence

**Comment by f1sh1918** [comment]
@connium since some designs are not ready or fully reviewed we could also continue with that issue that may not require a design.

We should just define if this will be an absence notice or a general note that will be added after the mail text

**Comment by connium** [comment]
I agree that we cannot tackle tasks for which there is no finished+reviewed design. So if you want to continue with tasks that are part of the Bavaria 2025 package, this is the right task.

Unfortunately I don't understand your proposed solution. I suggest that we have a short call about this.

**Comment by f1sh1918** [comment]
There is no clear description how it should look like.
Since we do not support styling yet, the question is if we provide a fixed "headline" and then include the text part that the region provides

I think if you just add a normal text i may be overseen if there is no highlighting @connium

**Comment by connium** [comment]
I checked how other services handle such messages, and this is how I would write it:

> Sehr geehrte/r {Vorname Nachname},
>
> wir haben Ihren Antrag auf die Bayerische Ehrenamtskarte erhalten.
>
> Den aktuellen Status Ihres Antrags können sie jederzeit unter folgendem Link einsehen. Dort haben Sie auch die Möglichkeit, Ihren Antrag bei Bedarf zurückzuziehen:
> {Link}
>
> Bitte beachten Sie, dass Antworten auf diese E-Mail werden nicht gelesen werden, da es sich um eine automatische Bestätigung handelt.
>
> Wenn Sie Fragen zu Ihrem Antrag haben, wenden Sie sich bitte direkt an Ihr zuständiges Landratsamt oder die Verwaltung Ihrer kreisfreien Stadt.
>
> {Abwesenheitsnotiz}
>
> Mit freundlichen Grüßen
> \- Ehrenamtskarte-Bayern-Verwaltung
